src/Aspire.Hosting.Azure.AppService/AzureAppServiceWebsiteContext.cs

```diff
@@ -224,6 +224,9 @@ public void BuildWebSite(AzureResourceInfrastructure infra)
                 LinuxFxVersion = "SITECONTAINERS",
                 AcrUserManagedIdentityId = acrClientIdParameter,
                 UseManagedIdentityCreds = true,
+                // Setting NumberOfWorkers to maximum allowed value for Premium SKU
+                // https://learn.microsoft.com/en-us/azure/app-service/manage-scale-up
+                NumberOfWorkers = 30,
```

Comment:
Can we add a comment explaining why we are doing this? It seems surprising that we would set this by defualt.